src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRunCodegenNodeFactory.cs

```diff
@@ -105,22 +105,15 @@ private ISymbolNode CreateReadyToRunHelperCell(ReadyToRunHelper helperId)
         }
 
         public IMethodNode MethodEntrypoint(
-            MethodDesc targetMethod,
-            TypeDesc constrainedType,
-            MethodDesc originalMethod,
-            ModuleToken methodToken,
+            MethodWithToken method,
             bool isUnboxingStub,
             bool isInstantiatingStub,
             SignatureContext signatureContext)
         {
-            bool isLocalMethod = CompilationModuleGroup.ContainsMethodBody(targetMethod, false);
-            if (targetMethod == originalMethod || isLocalMethod)
-            {
-                constrainedType = null;
-            }
+            bool isLocalMethod = CompilationModuleGroup.ContainsMethodBody(method.Method, false);
```

Comment:
Now there's only one use of `isLocalMethod` we could just replace its use below with `CompilationModuleGroup.ContainsMethodBody(method.Method, false)` in the if condition.

Reply:
Good point, thanks for pointing that out; fixed in 2nd commit.